gtsam/navigation/NavState.cpp

```diff
@@ -103,6 +103,16 @@ bool NavState::equals(const NavState& other, double tol) const {
       && equal_with_abs_tol(v_, other.v_, tol);
 }
 
+//------------------------------------------------------------------------------
+#ifdef GTSAM_EXTENDED_POSE_RETRACT
```

Comment:
I think this is done for the ImuFactor only, hence I’d prefer a solution that chooses NavState::exmap in the IMUFactor itself, once the other PR lands, if LieGroup integration is selected.